src/services/news/index.js

```diff
@@ -1,5 +1,7 @@
+const path = require('path');
```

Comment:
Codacy found an issue: [File has too many classes (2). Maximum allowed is 1.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6067861)